app.rb

```diff
@@ -1,9 +1,31 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require 'pg'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
```

Comment:
It's good to remove any extraneous or redundant code from your implementation, as a quick win for a cleanly written implementation.